bin/main.ml

```diff
@@ -114,13 +114,19 @@ let rec draw_letters ctx lst =
           (Zed_string.of_utf8 (String.make 1 h))
       else ()
 
-let draw_players ctx player =
-  let name_points =
-    String.concat " : " [ player.name; string_of_int player.points ]
-  in
-  let ctx_size = LTerm_draw.size ctx in
-  LTerm_draw.draw_string_aligned ctx (ctx_size.rows / 8) H_align_center
-    (Zed_string.of_utf8 name_points)
+let rec draw_players ctx = function
+  | [] -> ()
+  | h :: t ->
+      if draw_players ctx t = () then
+        let name_points =
+          String.concat " : " [ h.name; string_of_int h.points ]
+        in
+        let ctx_size = LTerm_draw.size ctx in
+        LTerm_draw.draw_string_aligned ctx
+          ((4 - List.length t) * (ctx_size.rows / 8))
```

Comment:
I used `7 - List.length t` and `(ctx_size.rows / 8)` in `draw_letters` based on the number of letters. I suggest the same happen here otherwise it can be drawn/formatted weirdly based on terminal size.

Suggestion:
Change line 117 `lec rec draw_players ctx = function` to 
` let rec draw_players ctx num_players = function `

and line 120 ` if draw_playeres ctx t = () then ` to
` if draw_players ctx num_players t = () then `

and Line 126 `((4 - List.length t) * (ctx_size.rows / 8))` to 
` (num_players - List.length t) * (ctx_size.rows / (num_players + 1)) ` 
Note: I use `num_players + 1` because just `num_players` sometimes causes one of the players to be drawn out of frame. If this doesn't work for most terminal sizes try playing around with this.